gui/GameConqueror.py

```diff
@@ -1040,9 +1039,26 @@ def check_backend_version(self):
         if self.backend.get_version() != VERSION:
             self.show_error(_('Version of scanmem mismatched, you may encounter problems. Please make sure you are using the same version of GameConqueror as scanmem.'))
 
-    
 
 if __name__ == '__main__':
+    # Parse cmdline arguments
+    parser = argparse.ArgumentParser(prog='GameConqueror',
+                                     description="A GUI for scanmem, a game hacking tool"
+
+                                    )
+    parser.add_argument('-v', '--version', action='version', version='%(prog)s ' + VERSION)
```

Comment:
It doesn't make much sense to require root privileges to print a version number or the help text. So maybe also the gameconqueror script should be modified. At least openSUSE won't require root privileges at all as they don't have the YAMA security module and don't want to set the `ptrace_scope` to 1.